app/controllers/stores_controller.rb

```diff
@@ -5,6 +5,8 @@ def index
   end
 
   def show
-    @store = Store.find(params[:id])
+    unless @store = Store.find_by(slug: params[:store_slug])
+      render file: "/public/404"
+    end
   end
```

Comment:
Can you explain this part to me tomorrow? I don't understand what the unless is doing.